.github/workflows/deploy-storefrontcloud.yml

```diff
@@ -16,7 +16,8 @@ jobs:
         with:
           node-version: "12.x"
       - name: Build and publish docker image
-        uses: elgohr/Publish-Docker-Github-Action@master
+        # 3.04 is hardcoded as a workaround for https://github.com/elgohr/Publish-Docker-Github-Action/issues/134
```

Comment:
I think you can keep it as a comment
`# uses: elgohr/Publish-Docker-Github-Action@master`